javers-core/src/main/java/org/javers/core/metamodel/clazz/ShallowReferenceDefinition.java

```diff
@@ -0,0 +1,14 @@
+package org.javers.core.metamodel.clazz;
+
+import org.javers.core.metamodel.type.EntityType;
+
+/**
+ * Recipe for {@link EntityType}.
+ *
+ * @author akrystian
+ */
+public class ShallowReferenceDefinition extends EntityDefinition {
```

Comment:
Hmm. To jest raczej do rozróżnienia ManagedClassFactory.create(ClientsClassDefinition clientsClassDefinition) obiektu który ma zostać stworzony. To jest nadal encja ale konstruowana z wszystkimi parametrami filtrowanymi (z wyjątkiem Id).